bay/images/Dockerfile.clamav

```diff
@@ -6,6 +6,7 @@ RUN apk add --no-cache -u --repository http://dl-cdn.alpinelinux.org/alpine/edge
   clamav-daemon \
   clamav-libunrar \
   unrar \
+  py3-setuptools \
```

Comment:
An update to `supervisor` has removed the installation of this dependency and as a result it was failing the build.